### PR TITLE
Document sort option

### PR DESCRIPTION
**Pull request by valeriangalliat**
Must be merged in both `master` and `develop`; the feature is already published.

### CODE REVIEW

File: customising-the-view/index.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This is done exactly like the regular configuration, within the same configurati
 | `shortcutIcon`      | String  | `""`                    |
 | `googleAnalytics`   | String  | `""`                    |
 | `trackingCode`      | String  | `""`                    |
+| `sort`              | Arrat   |                         |
```
Comment (HugoGiraudel) [inline customising-the-view/index.md]: `Array` not `Arrat`.

File: customising-the-view/index.md [hunk 2]
```diff
@@ -94,6 +95,29 @@ trackingCode: |
 
 <p class="note  note--danger"><strong>Beware!</strong> This is a possible door to JavaScript code injection. Be careful.</p>
 
+## Sort
+
+You can customize the order of your items with this option.
+
+Supported criteria are:
+
+* `group`
```
Comment (HugoGiraudel) [inline customising-the-view/index.md]: Add a link to http://sassdoc.com/configuration/#groups.

File: customising-the-view/index.md [hunk 3]
```diff
@@ -94,6 +95,29 @@ trackingCode: |
...
+* `file`
+* `line`
+* `access`
```
Comment (HugoGiraudel) [inline customising-the-view/index.md]: Add a link to http://sassdoc.com/annotations/#access.

File: customising-the-view/index.md [hunk 4]
```diff
@@ -94,6 +95,29 @@ trackingCode: |
...
 
+## Sort
+
+You can customize the order of your items with this option.
+
+Supported criteria are:
+
+* `group`
+* `file`
+* `line`
```
Comment (HugoGiraudel) [inline customising-the-view/index.md]: Consider a link to http://sassdoc.com/annotations/#comment-range?

File: customising-the-view/index.md [hunk 5]
```diff
@@ -94,6 +95,29 @@ trackingCode: |
...
+the last character is either `>` or `<` (respectively desc and asc).
+
+{% highlight yaml %}
+sort:
+  - access
+  - line>
+  - group
+  - file
+{% endhighlight %}
+
```
Comment (HugoGiraudel) [inline customising-the-view/index.md]: Explain in plain English how the given example sorts the data.

**Comment by valeriangalliat** [comment]
Thank you, updated.
